iam_groups_authn/sync.py

```diff
@@ -307,18 +289,22 @@ async def revoke_iam_group_role(
         role: IAM group role.
         users_with_roles_future: Future for list of database users who have group role.
         iam_users_future: Future for list of IAM users in IAM group.
+        database_type: Type of database.
     """
     # await dependent tasks
     iam_users = await iam_users_future
     users_with_roles = await users_with_roles_future
```

Comment:
nit: can we await these together? That way if the second one fails first we don't block on it